Middleware/DebugMiddleware.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Doctrine\Bundle\DoctrineBundle\Middleware;
+
+use Doctrine\DBAL\Driver as DriverInterface;
+use Doctrine\DBAL\Driver\Middleware;
+use Symfony\Bridge\Doctrine\Middleware\Debug\DebugDataHolder;
+use Symfony\Bridge\Doctrine\Middleware\Debug\Driver;
+use Symfony\Component\Stopwatch\Stopwatch;
+
+class DebugMiddleware implements Middleware, ConnectionNameAwareInterface
```

Comment:
why aren't you using the DebugMiddleware defined in Symfony ?

Reply:
Because I need the  `ConnectionNameAwareInterface` in the bundle. Currently it's how the bundle is designed, one middleware (or previously a DebugStack) by connection. It was not a requirement in the bridge where the middleware sat, so by default the middleware doesn't implement `ConnectionNameAwareInterface`.